crates/ra_assists/src/split_import.rs

```diff
@@ -10,10 +10,7 @@ pub(crate) fn split_import(mut ctx: AssistCtx<impl HirDatabase>) -> Option<Assis
     let path = ast::Path::cast(colon_colon.parent())?;
     let top_path = successors(Some(path), |it| it.parent_path()).last()?;
 
-    let use_tree = top_path.syntax().ancestors().find_map(ast::UseTree::cast);
-    if use_tree.is_none() {
-        return None;
-    }
+    let _use_tree = top_path.syntax().ancestors().find_map(ast::UseTree::cast)?;
```

Comment:
The original version seems a touch more readable to me

Reply:
Fixed